donut/templates/includes/header.html

```diff
@@ -28,12 +28,11 @@
                         <ul class="dropdown-menu" role="menu">
                             <li><a href="{{ url_for('feedback.feedback', group='arc') }}">ARC Feedback</a></li>
                             <li><a href="{{ url_for('calendar.calendar') }}">Calendar</a></li>
-                            <li><a href="#">Course Reviews</a></li>
                             <li><a href="{{ url_for('courses.planner') }}">Course Planner</a></li>
                             <li><a href="{{ url_for('courses.scheduler') }}">Course Scheduler</a></li>
                             <li><a href="{{ url_for('directory_search.directory_search') }}">Directory</a></li>
                             <li><a href="{{ url_for('feedback.feedback', group='donut') }}">Donut Feedback</a></li>
-			    <li><a href=" {{url_for('marketplace.marketplace') }} ">Marketplace</a></li>
+			                <li><a href=" {{url_for('marketplace.marketplace') }} ">Marketplace</a></li>
```

Comment:
Let's standardize the indentation to use only spaces here so this lines up better